knowledge_repo/app/routes/comment.py

```diff
@@ -35,9 +35,13 @@ def post_comment():
     if not post:
         raise Exception('Unable to find post')
 
-    comment = (db_session.query(Comment)
-                         .filter(Comment.id == comment_id)
-                         .first())
+    if comment_id is not None:
+        comment = (db_session.query(Comment)
```

Comment:
Ah, I agree. I was being cautious for `comment_id`s with the value zero, but that probably doesn't happen!

Updated!